lib/internal/assert/myers_diff.js

```diff
@@ -84,8 +93,10 @@ function backtrack(trace, actual, expected, checkCommaDisparity) {
     const prevY = prevX - prevDiagonalIndex;
 
     while (x > prevX && y > prevY) {
-      const value = !checkCommaDisparity ||
-        StringPrototypeEndsWith(actual[x - 1], ',') ? actual[x - 1] : expected[y - 1];
+      const value =
+        !checkCommaDisparity || StringPrototypeEndsWith(actual[x - 1], ',') ?
+          actual[x - 1] :
+          expected[y - 1];
```

Comment:
```suggestion
      const actualItem = actual[x - 1];
      const value =
        !checkCommaDisparity || StringPrototypeEndsWith(actualItem, ',') ?
          actualItem :
          expected[y - 1];
```

We can exctract into a variable? (I used a random name you can probably use a better one 😆 )